lib/par_incr.ml

```diff
@@ -1,6 +1,8 @@
+let null = Obj.magic (ref ())
```

Comment:
I'd move this inside the `Var` module, because it should not be used outside of it.

Reply:
Ok I'll do that